src/tests/Button.test.js

```diff
@@ -3,12 +3,31 @@ import ReactDOM from 'react-dom';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import Button from '../components/Button/Button';
+import App from '../App';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe('<Button />', () => {
-  it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Button />, div);
+describe('Button component', () => {
+  it('shallow renders without crashing', () => {
+      shallow(<Button />);
   });
-});
+
+  it("it render an button", () => { 
+      const button = shallow(<Button />);
+      const div1 = button.find('div')
+      expect(div1.length).toEqual(1)
+  });
+
+  it('test for finding if button is clicked', () => {
+    const snake = jest.fn(()=> {});
+    const button = shallow(<Button clickHandler={snake} />);
+    const instance = button.instance();
+
+    instance.handleClick();
```

Comment:
I like how you call the handleClick function from the Button Component in here, but consider simulating a click on the button to ensure it fires properly when clicked as well:` button.find('button').simulate('click')`
Refer to this documentation here for more insight: https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/simulate.md